### DISCUSSION TITLE
How are we planning to maintain the documentation site?

### DISCUSSION THREAD

**Discussion by xmflsct** [mastodon/mastodon#18216] [General]
I am a third-party Mastodon app developer. Since the release of `v3.5.0`, the additions and changes are not reflected in the documentation, and that I need to figure out the changes by looking at each PR which is not ideal. I hope third-party apps are still as important as the official apps?

**Comment by markcellus** [comment]
Looks like the changelog appears on the [releases page](https://github.com/mastodon/mastodon/releases). Or were you thinking of something else?

**Reply by xmflsct** [reply]
Yes there are, but what about the official [documentation website](https://docs.joinmastodon.org/methods/statuses/)? The release page does not cover all the details that is needed for a proper third-party app development.

**Reply by markcellus** [reply]
Oh I see. The API docs should be updated upon new releases. Hopefully the Mastodon contributors will see this and can address. :+1:

**Comment by xmflsct** [comment]
Bump
